### PR TITLE
Bugfix/compilation on vs2019

### PR DESCRIPTION
**Pull request by alt-graph**
This PR fixes compilation for VS2019 installations on English-speaking Windows installations. It also adds a few tiny corrections to the README.

### CODE REVIEW

File: tools/make_vs.bat [hunk 1]
```diff
@@ -106,20 +106,24 @@ ninja -C "%REPO_ROOT%%FOLDER%" %TARGET%
...
-            set VC_YEAR=2017\
-        ) ELSE (
-            echo "Can not find '%VC_PREFIX%2017' or '2019'"
-            exit /B 1
-        )
+        echo "Visual Studio base directory not found"
+        exit /B 1
+    )
+	
+	@IF EXIST "%VC_PREFIX%2022" (
```
Comment (Finii) [inline tools/make_vs.bat:118]:
```suggestion
@IF EXIST "%VC_PREFIX%2022" (
```

or similar

File: tools/make_vs.bat [hunk 2]
```diff
@@ -106,20 +106,24 @@ ninja -C "%REPO_ROOT%%FOLDER%" %TARGET%
...
+        set VC_YEAR=2022\
+    ) ELSE IF EXIST "%VC_PREFIX%2019" (
+		set VC_YEAR=2019\
+	) ELSE IF EXIST "%VC_PREFIX%2017" (
```
Comment (Finii) [inline tools/make_vs.bat:122]: Hmm, again switch in indentation.
tabs versus spaces?

**Review by Finii — CHANGES_REQUESTED** [review]

**Comment by Finii** [comment]
Probably we should aim to change this to powershell at some point, my eyes glace over when looking at the code 😬

File: tools/make_vs.bat (hunk 2 above)
Comment (alt-graph) [reply tools/make_vs.bat]: Uh, sorry. Yes, tabs... that was an editor I do not use regularly. Force-pushed to correct the indentation.

File: tools/make_vs.bat (hunk 1 above)
Comment (alt-graph) [reply tools/make_vs.bat]: Fixed.

**Review by Finii — APPROVED** [review]
Very good documentation fixes, btw

**Pull request merged**